src/components/Grants/BannerItem.css

```diff
@@ -5,6 +5,11 @@
   justify-content: space-between;
 }
 
+.GrantsBannerItem--noUrl {
```

Comment:
Maybe doing the opposite and changing this to `.GrantsBannerItem--clickable`?

Reply:
The default behavior is clickable, I need it to be non-clickable 😅